cdragontoolbox/wad.py

```diff
@@ -231,6 +242,7 @@ def extract(self, output, overwrite=True):
         logger.info(f"extracting {self.path} to {output}")
 
         self.set_unknown_paths("unknown")
+        self.sanitize_paths()
```

Comment:
Exporters don't call `extract()` directly so they won't use sanitized paths.
You probably want to call `sanitize_path()` in `Exporter.add_path()` (after call to `wad.guess_extensions()`).